.github/workflows/ci.yml

```diff
@@ -36,10 +36,37 @@ jobs:
         go install golang.org/x/tools/cmd/goimports@latest
         go install github.com/mattn/goveralls@latest
     - if: ${{ runner.os == 'Linux' }}
-      run: if [ $(goimports -l .) ]; then goimports -d .; echo 'Failed the goimports format check. Please format the code using "goimports -w ."'; exit 1; fi
-    - run: go test -mod=mod -coverpkg="./..." -coverprofile=covprofile ./...
-    - env:
+      env:
         COVERALLS_TOKEN: ${{ secrets.github_token }}
-      run: goveralls -coverprofile=covprofile -service=github -ignore="confgenerator/filter/internal/generated/*/*"
+      run: |
+        if [ $(goimports -l .) ]; then goimports -d .; echo 'Failed the goimports format check. Please format the code using "goimports -w ."'; exit 1; fi
+        go test -mod=mod -coverpkg="./..." -coverprofile=covprofile ./...
+        goveralls -coverprofile=covprofile -service=github -ignore="confgenerator/filter/internal/generated/*/*"
+    - id: 'set-coverage-vars'
+      if: ${{ runner.os == 'Linux' }}
+      name: 'Set Coverage Env Vars'
+      run: echo "ABS_DIR=ng3-customer-telemetry-coverage/absolute/cloud-ops-agent/${{ github.run_id }}" >> "$GITHUB_ENV"
+    - id: 'prepare-coverage-data'
+      if: ${{ runner.os == 'Linux' }}
+      name: 'Prepare Coverage Data'
+      run: |
+        mkdir -p "${{ env.ABS_DIR }}"
+        echo '{"host":"github","project":"cloud-ops-agent","trace_type":"LCOV","commit_id":"${{ github.sha }}","ref":"HEAD","source":"https://github.com/GoogleCloudPlatform/ops-agent","owner":"stackdriver-instrumentation-dev","bug_component":"931151"}' > ${{ env.ABS_DIR }}/metadata.json
```

Comment:
Is `https://github.com/GoogleCloudPlatform/ops-agent` not `${{ github.server_url }}/${{ github.repository }}` (modulo my other comment about forks)? Or can this even be `${{ github.repositoryUrl }}` (https://docs.github.com/en/actions/learn-github-actions/contexts#github-context)?

Reply:
nice suggestion, done